docs/cloud/azureai/runtime-change-log.md

```diff
@@ -0,0 +1,20 @@
+# Change log of default runtime image
+
+## 20240116.v1
```

Comment:
how do user do with this version? may add some introduction ?

Reply:
Will do.

Reply:
I added some description of runtime. Also mention how to check runtime version and pull runtime image.